packages/language-server/src/completion-items.ts

```diff
@@ -167,6 +166,26 @@ function createTextEdits(
       }
       break;
     }
+    case "UI5PropsInXMLAttributeKey": {
+      range = getXMLAttributeKeyRange(suggestion.astNode) ?? range;
+
+      // Auto-insert ="<default value>" (or ="" if there is no default value) for property attributes
+      if (suggestion.astNode.syntax.value === undefined) {
+        const defaultValue = suggestion.ui5Node.default;
+        // Only simple values are added in the completion (there are default values which are objects, arrays and null)
+        if (
+          defaultValue !== undefined &&
+          includes(["number", "boolean", "string"], typeof defaultValue)
+        ) {
+          // The ${0:} makes the text between : and } selected
```

Comment:
Can you please use the LSP term in the comment so it would be easy to look for in the specs?
e.g:
> // The LSP **PlaceHolder** `${0:}` makes the text between : and } selected

Maybe even focus on the **why** (functional scenario) more and let future developers reference the LSP spec 
for **how** exactly to use this syntax.
> // The LSP **PlaceHolder** `${0:}` selects text after completion for easy editing if the default value is not needed by the user.

Reply:
Done